tcl/is.tcl

```diff
@@ -534,10 +266,58 @@ namespace eval qc::is {
         return [expr {[string toupper $bool] in {Y N YES NO TRUE FALSE T F 0 1}}]
     }
 
-    proc decimal {number} {
```

Comment:
Anything that is valid in Postgresql should be valid here.
Our data types and literal representation should correspond as closely as possible with pg.